docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -52,13 +52,64 @@ The above ensures that the Service Locator knows about the `weather` service. Th
 
 Note that if the service you want to communicate with is actually a Lagom service, you may want to read the documentation for [[integrating with an external Lagom projects|MultipleBuilds]].
 
+
+# Service Gateway
```

Comment:
I wrote the comment above before reading down and seeing the start & stop information, and now I think that it could be awkward to split them, too. Maybe rename the page title to "Service Locator and Service Gateway" to make it more clear that we're talking about both things on one page?